Lib/logging/__init__.py

```diff
@@ -191,10 +191,11 @@ def _checkLevel(level):
         rv = level
     elif str(level) == level:
         if level not in _nameToLevel:
-            raise ValueError("Unknown level: %r" % level)
+            raise ValueError(f"Unknown level: {repr(level)}")
```

Comment:
To get to this code you would need to have passed the test `str(level) == level`, how would a tuple do this? I wouldn't think this line needs changing.